dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -269,7 +270,10 @@ private String resultProcess(ResultSet resultSet) throws Exception {
                 logger.info("row {} : {}", i + 1, row);
             }
         }
-        String result = JSONUtils.toJsonString(resultJSONArray);
+
+        String result = resultJSONArray.isEmpty() ?
```

Comment:
Is it OK if return the empty row? Because it will deal out param if define the param pass.
see https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/guide/parameter/context.html

Reply:
I tested it according to the document, such as
<img width="536" alt="image" src="https://user-images.githubusercontent.com/35831367/191460410-f5c84b86-19fa-43cb-af22-3f9751839560.png">
<img width="477" alt="image" src="https://user-images.githubusercontent.com/35831367/191460507-3ea01a6b-7a18-4908-9564-97a4732bf1ea.png">

Reply:
This is the normal case with data
<img width="354" alt="image" src="https://user-images.githubusercontent.com/35831367/191461966-1e0fb4f4-7ded-4864-bbc5-6b75e65eeae8.png">